src/commands/api-mesh/__tests__/describe.test.js

```diff
@@ -173,6 +180,11 @@ describe('describe command tests', () => {
 	});
 
 	test('should succeed if valid details are provided', async () => {
+		let fetchedMeshConfig = sampleCreateMeshConfig;
+		fetchedMeshConfig.meshConfig.meshId = 'dummy_id';
+		fetchedMeshConfig.meshConfig.meshURL = 'https://tigraph.adobe.io';
```

Comment:
Is this test meant for a TI url? It looks like we created a new one below for TI and this one should be testing Non-TI?

Reply:
The tests are swapped